aiven/client/argx.py

```diff
@@ -122,12 +122,12 @@ def save(self):
 
 
 class CommandLineTool:  # pylint: disable=old-style-class
-    def __init__(self, name):
+    def __init__(self, name, parser: argparse.ArgumentParser = None):
```

Comment:
This should have been Optional if the default is None.

Reply:
Ugh good point. I'll submit another PR.